source/extensions/filters/http/local_ratelimit/local_ratelimit.cc

```diff
@@ -91,6 +91,12 @@ Http::FilterHeadersStatus Filter::decodeHeaders(Http::RequestHeaderMap& headers,
     populateDescriptors(descriptors, headers);
   }
 
+  for (const auto& request_descriptor : descriptors) {
+    for (const Envoy::RateLimit::DescriptorEntry& entry : request_descriptor.entries_) {
+      ENVOY_LOG(debug, "populate descriptors: key={} value={}", entry.key_, entry.value_);
```

Comment:
It does introduce extra time only useful for debugging, isn't it?

Reply:
Yeah I would wrap the entire loop in whether debug logging is enabled or not.

Reply:
sounds good, I will update it later.

Reply:
updated, PTAL.